internalsdk/ios/ios.go

```diff
@@ -190,28 +195,25 @@ func (c *iosClient) start() (ClientWriter, error) {
 		return nil, log.Errorf("error loading user config: %v", err)
 	}
 	log.Debugf("Running client at config path '%v'", c.configDir)
+	start := time.Now()
+	log.Debugf("User config process start at %v", start)
 	dialers, err := c.loadDialers()
 	if err != nil {
 		return nil, err
 	}
+	if len(dialers) == 0 {
+		return nil, errors.New("No dialers found")
+	}
 	tracker := stats.NewTracker()
 	dialer, err := bandit.NewWithStats(dialers, tracker)
 	if err != nil {
 		return nil, err
 	}
 	go func() {
 		tracker.AddListener(func(st stats.Stats) {
```

Comment:
@myleshorton question to you

What is supposed to be a time when we create `bandit.NewWithStats` and we will get updated from stats?

Looking at my logs sometimes it takes around 2 min to get a callback

```
May 24 10:43:27.286 - 0m12s DEBUG ios: ios.go:199 User config process start at 2024-05-24 10:43:27.286492 +0000 UTC m=+14.450121584

May 24 10:45:57.004 - 2m42s DEBUG ios: ios.go:215 Stats update at 2024-05-24 10:45:57.003982 +0000 UTC m=+164.167602251
```
cc @atavism

Reply:
> What is supposed to be the time when we create bandit.NewWithStats and when we will get updated from stats? is there any timeout?

bandit should send updated stats every time there's a successful dial: https://github.com/getlantern/flashlight/blob/main/bandit/bandit.go#L161